sdk/src/main/scala/com/stratio/sparkta/sdk/Output.scala

```diff
@@ -172,8 +182,11 @@ abstract class Output(keyName: String,
 
 object Output {
 
-  final val SEPARATOR = "_"
-  final val ID = "id"
+  final val Separator = "_"
+  final val Id = "id"
+  final val FixedAggregation = "fixedAggregation"
```

Comment:
Aggregation and multiplexer maybe aren't related to the output, I think that we need to move this values to other class.